js_basics_array/task3.js

```diff
@@ -0,0 +1,4 @@
+const array = [10, 20, 30, 40, 50, -10, 0, 378];
+let sum = array.reduce((accumulator, currentElement) => accumulator + currentElement);
```

Comment:
declare initial value for accumulator explicitly, it will help you to write more predictable code
```js
array.reduce((accumulator, currentElement) => accumulator + currentElement,0)
```